src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceArea/ResourceArea.ts

```diff
@@ -289,6 +291,10 @@ export class Service implements AdhTopLevelState.IAreaInput {
         });
     }
 
+    public getName(resourceType : string) : string {
+        return this.provider.names[resourceType];
+    }
```

Comment:
Another argument for moving this to another place is a circular dependency:

ResourceArea -> ProcessListing -> Listing -> ResourceArea (names)